tools/testdxildll/DxilLib.cpp

```diff
@@ -0,0 +1,150 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Would you mind renaming DxilLib.* files to TestDxilLib.*? I am often searching through the whole code base and since this code is similar to the actual DxilLib.cpp it would be nice to have it under a different name to it can be clearly seen in the search results that this is the test dll.

Reply:
Will do